CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Removed
 - Utils functions chunked and flatten_rows_dict in getl/common/upsert.py
 
+### Fixed
+- When checking if a file registry exists on an s3 path or on a directory that exists but is empty, a different exception is raised
```

Comment:
The correct naming is prefix and not directory on S3
"When checking if a file registry exists on an S3 path or on a prefix that exists but is empty"

Reply:
I meant a directory as in ”/home/ec2-user/” or an s3 path like ”s3://bucket/prefix/to/file”